core/src/main/java/edu/wpi/grip/core/events/ExceptionEvent.java

```diff
@@ -22,7 +23,7 @@
      * @param message   The message associated with this event.
      *                  If <tt>null</tt> will use {@link Exception#getMessage()}
      */
-    public ExceptionEvent(Object origin, Exception exception, String message) {
+    public ExceptionEvent(Object origin, @Nullable Exception exception, @Nullable String message) {
```

Comment:
Because I added a google sanity test to the package. It was either fix the code or disable it in the tests. I decided to fix it in the code.